src/Constants.ts

```diff
@@ -4,7 +4,8 @@ import path = require('path');
 export const PMD_VERSION = '6.44.0';
 export const CATALOG_FILE = 'Catalog.json';
 export const CUSTOM_PATHS_FILE = 'CustomPaths.json';
-export const CONFIG_FILE = 'Config.json';
+export const CONFIG_FILE = 'Config-v3.json';
```

Comment:
Can we call these "CONFIG_FILE" and "CONFIG_V3_FILE"? 2.x is still our GA version and hasn't become a legacy. Also, based on the explanation below that we could go back to a single config file when v3 becomes `latest`, we wouldn't want to preemptively call it legacy.

Reply:
My logic for `CONFIG_FILE` and `LEGACY_CONFIG_FILE` was that it was easier to change the value of the constant than rename the constant and change all of the places where it's used. But that's a good point, so I'll change it.